R/write_pxl_file.R

```diff
@@ -119,6 +119,13 @@ WriteMPX_pxl_file <- function (
 
   # Create a temporary directory
   pxl_folder <- file.path(fs::path_temp(), .generate_random_string())
+  while (fs::dir_exists(pxl_folder)) {
+    err <- try(fs::dir_delete(pxl_folder))
+    if (inherits(err, "try-error")) {
+      warn(glue("Failed to delete temporary directory '{col_br_blue(pxl_folder)}'."))
+      pxl_folder <- file.path(fs::path_temp(), .generate_random_string())
+    }
+  }
```

Comment:
Here we are doing the same with the temporary folder, right? Yes, I think it would be nice to package it in a utility function.

Reply:
I think that's a good idea!

Reply:
Added an internal function `.create_unique_temp_dir` to `utils.R` that replaces these code blocks.